third_party/amd/python/test/test_extract_slice_concat_op.py

```diff
@@ -41,48 +41,86 @@ def __str__(self):
 # -----------------------
 # test extract slice
 # -----------------------
-
 regs2x2 = [[1, 0], [0, 1]]
+
 lanes8x8 = [[2, 0], [4, 0], [8, 0], [0, 2], [0, 4], [0, 8]]
-warps2x2 = [[16, 0], [0, 16]]
-redundant_ll = LinearLayout([[0, 0]] + regs2x2, lanes8x8, warps2x2, block=[])
-non_redundant_ll = LinearLayout(regs2x2, lanes8x8, warps2x2, block=[])
+warps2x2_64 = [[16, 0], [0, 16]]
+redundant_ll = LinearLayout([[0, 0]] + regs2x2, lanes8x8, warps2x2_64, block=[])
+non_redundant_ll = LinearLayout(regs2x2, lanes8x8, warps2x2_64, block=[])
+
+lanes8x4 = [[2, 0], [4, 0], [8, 0], [0, 2], [0, 4]]
+warps2x2_32 = [[16, 0], [0, 8]]
+redundant_ll_gfx1250 = LinearLayout([[0, 0]] + regs2x2, lanes8x4, warps2x2_32, block=[])
```

Comment:
Rather than naming these variables with suffix _gfx1250, let's label them with _32 and _64. Afterwards we choose 32 vs 64 based on arch.